### PR TITLE
add commands of how to delete a training job

### PR DESCRIPTION
**Pull request by ChanYiLin**
I added commands of how to delete a training job to the user guide for users who are not familiar with the Ksonnet.
I also added commands of how to monitor a tf-cnn jobs since I did not realize at first that tf-cnn job is also a tfjobs.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/378)

### CODE REVIEW

File: user_guide.md [hunk 1]
```diff
@@ -271,6 +271,12 @@ To run your job
 ks apply ${KF_ENV} -c ${JOB_NAME}
 ```
 
+To delete your job
```
Comment (lluunn) [inline user_guide.md]: I think delete should be after monitor.

File: user_guide.md [hunk 2]
```diff
@@ -290,6 +296,18 @@ Submit it
 ks apply ${KF_ENV} -c ${CNN_JOB_NAME}
 ```
 
+Delete it
+
+```
+ks delete ${KF_ENV} -c ${CNN_JOB_NAME}
+```
+
+Monitor it (Noted that tf-cnn job is also a tfjobs.)
```
Comment (lluunn) [inline user_guide.md]: Move it above delete too. And link to the doc  https://github.com/kubeflow/tf-operator#monitoring-your-job

**Comment by ChanYiLin** [comment]
I fixed the order of commands.
/assign @lluunn

**Pull request merged**